diff --git a/src/protocols/ssh/Makefile.am b/src/protocols/ssh/Makefile.am
index da8d1ea..e2e3ad4 100644
--- a/src/protocols/ssh/Makefile.am
+++ b/src/protocols/ssh/Makefile.am
@@ -70,7 +70,7 @@ noinst_HEADERS =                \
     terminal_handlers.h         \
     types.h
 
-libguac_client_ssh_la_CFLAGS = -Werror -Wall -pedantic -Iinclude @PANGO_CFLAGS@ @PANGOCAIRO_CFLAGS@ @LIBGUAC_INCLUDE@
+libguac_client_ssh_la_CFLAGS = -Werror -Wall -Iinclude @PANGO_CFLAGS@ @PANGOCAIRO_CFLAGS@ @LIBGUAC_INCLUDE@
 libguac_client_ssh_la_LIBADD = @LIBGUAC_LTLIB@
 libguac_client_ssh_la_LDFLAGS = -version-info 0:0:0 @SSH_LIBS@ @PTHREAD_LIBS@ @PANGO_LIBS@ @PANGOCAIRO_LIBS@ @CAIRO_LIBS@
 
